app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md

```diff
@@ -42,14 +42,21 @@ calls_to_action:
       - Cost: "Up to $earlyyears_fees_undergrad$ per year"
         Salary: "No"
         Length of training: "3 or 4 years"
-        Qualification: "Early years teacher status"
+        Qualification: "Early years teacher status and a bachelor’s degree"
   EY-self-funded:
     name: feature_table
     arguments:
       - Cost: "This could be around $earlyyears_fees_selffundedlow$ or $earlyyears_fees_selffundedhigh$ depending on the provider"
         Salary: "No"
         Length of training: "3 months"
         Qualification: "Early years teacher status"
+  EY-apprenticeship:
+    name: feature_table
+    arguments:
+      - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ are covered through the employer’s apprenticeship levy payments or co-funded with the government contributing 95% of the cost and employers contributing 5% of the cost. You do not need to do anything to access this funding."
```

Comment:
For other fees on this page, we say per year - worth checking with the team as to whether we can do the same here for consistency? Otherwise the fee looks huge in comparison!